crates/libm-test/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "libm-test"
+version = "0.1.0"
+authors = ["Benjamin Schultzer <benjamin@schultzer.com>"]
+
+[dependencies]
+libm = { version = "0.*", path = "../../" }
```

Comment:
I think this may want to unconditionally enabled the `checked` feature perhaps? Or reexport it so we can test with/without the feature?

Reply:
I've reexported the feature, it still unclear to me how this feature works? and what part of the code base it affects.